src/Files.App/Helpers/FilePropertiesHelpers.cs

```diff
@@ -15,11 +15,14 @@
 
 namespace Files.App.Helpers
 {
-	public static class FilePropertiesHelpers
+	public class FilePropertiesHelpers
```

Comment:
If the class is made non-static, it should be renamed `FilePropertiesHelper`, removing the "s" at the end.